src/lib/HederaLib.ts

```diff
@@ -51,4 +51,20 @@ export class HederaWallet {
       return { error: (e as Error).message }
     }
   }
+
+  public async signAndReturnTransaction(transaction: Transaction) {
+    try {
+      const signedTransaction = await transaction.sign(this.privateKey)
+      const signedTransactionBytes = signedTransaction.toBytes()
+      const encodedTransactionBytes = Buffer.from(signedTransactionBytes).toString('base64')
+      return {
```

Comment:
@HGraphPunks @hgraphql LMK what you think of the format here. Could also just be:
```js
{
  transactionBytes: encodedTransactionBytes
}
```

Reply:
I think this is cleaner for now. If we add type, which isn't defined currently, would you think transaction.bytes and tranaction.type is cleaner for expanding the json in the future

or is just adding transactionType in the root a solution that'd make it easier? I feel like there's no right or wrong, just a pattern to set.

Reply:
Good point, I pushed a change to include transaction type in the returned result. Now the params for the request and result are the same:
```js
{
  transaction: {
    type: string
    bytes: string
  }
}
```

Reply:
nice, yeah I think this looks clean